lib/src/builder/generation/impl_generation.dart

```diff
@@ -685,12 +690,17 @@ class ImplGenerator {
         ..writeln('//')
         ..writeln('// Implements constructor and backing map, and links up to generated component factory.')
         ..write(classDeclaration)
-        ..writeln('  $implName(Map backingMap) : this._props = backingMap ?? {};')
+        ..writeln('  // This initializer of `_props` to an empty map, as well as the reassignment')
+        ..writeln('  // of `_props` in the constructor body is necessary to work around an unknown ddc issue.')
+        ..writeln('  // See <https://jira.atl.workiva.net/browse/CPLAT-4673> for more details')
+        ..writeln('  $implName(Map backingMap) : this._props = {} {')
```

Comment:
Just thought of one thing; have you tried initializing to `null` instead? 

Would be nice to avoid this object creation if it's easy to; no big deal if it's a pain to try out, though.
```suggestion
        ..writeln('  $implName(Map backingMap) : this._props = null {')
```

Reply:
Unfortunately this doesn't work in ddc.